platform/rpc/nats/bus.go

```diff
@@ -321,6 +327,8 @@ func (n *NATS) Subscribe(instID flux.InstanceID, remote platform.Platform, done
 	}
 
 	go func() {
+		forceReconnect := time.NewTimer(maxAge)
+		defer forceReconnect.Stop()
```

Comment:
Yes, but the channels so-created won't get garbage collected until the timer fires, which might be two hours hence. Admittedly not a huge risk, but this avoids it at negligible expense.